crates/cloud/src/client.rs

```diff
@@ -196,25 +179,20 @@ impl Client {
         &self,
         id: Uuid,
         name: Option<String>,
-        domain: Option<String>,
         revision_selection_strategy: Option<ChannelRevisionSelectionStrategy>,
         range_rule: Option<String>,
         active_revision_id: Option<Uuid>,
-        certificate_id: Option<Uuid>,
         environment_variables: Option<Vec<UpdateEnvironmentVariableDto>>,
     ) -> anyhow::Result<()> {
         let command = PatchChannelCommand {
             channel_id: Some(id),
             name: name.map(|n| Box::new(StringField { value: Some(n) })),
-            domain: domain.map(|d| Box::new(StringField { value: Some(d) })),
             revision_selection_strategy: revision_selection_strategy
```

Comment:
Does ChannelRevisionSelectionStrategyField still exist on the new specification of the could swagger file (asking this because when I run locally the revision_selection_strategy is a string, as seen in the picture below)?
![image](https://user-images.githubusercontent.com/20836504/194876800-02a98cca-f347-4161-a890-9b5abea3b580.png)

Reply:
Yes

Reply:
https://github.com/fermyon/cloud-openapi/blob/350f67d7abf3cab9da83a44ba93ff5bfc2ad8b29/swagger.json#L1351-L1357

Reply:
Let me try to re-generate the openapi against main and see if I missed something.

Reply:
I just re-generated the new specification of the cloud swagger file and I received the same file as before.

As demonstrated in https://github.com/fermyon/spin/pull/794#discussion_r992673079, the type is a String, but it is still generating the "ChannelRevisionSelectionStrategyField" as before.

Reply:
https://github.com/fermyon/cloud-openapi/commit/350f67d7abf3cab9da83a44ba93ff5bfc2ad8b29 demonstrates what changed between updates.